.travis.yml

```diff
@@ -19,6 +19,8 @@ before_install:
   - pip install --no-cache-dir -U -r requirements-test.txt
   - npm install jslint -g
   - jslint django_netjsongraph/static/netjsongraph/js/*.js
+  # to be removed after release of openwisp-utils 0.4.5
+  - pip install --upgrade https://github.com/nepython/openwisp-utils/tarball/master
```

Comment:
this is not needed anymore now

Reply:
Extremely sorry forgot about it :sweat_smile: